components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="content-container">
+  <div class="container">
+    <main>
+      <chef-breadcrumbs>
+        <chef-breadcrumb [link]="['/infrastructure/chef-servers']">Chef Servers</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', org?.server_id]">Orgs</chef-breadcrumb>
+        {{ org?.name }}
+      </chef-breadcrumbs>
+      <chef-page-header>
+        <chef-heading>{{ org?.name }}</chef-heading>
+        <table>
+          <thead>
+            <tr class="detail-row">
+              <th class="id-column">Name</th>
+              <th class="id-column">Admin User</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="detail-row">
+              <td class="id-column">{{ org?.name }}</td>
+              <td class="id-column">{{ org?.admin_user }}</td>
+            </tr>
+          </tbody>
+        </table>
+        <chef-tab-selector [value]="tabValue" (change)="onSelectedTab($event)">
+        <chef-option value='cookbooks' data-cy="cookbooks-tab">Cookbooks</chef-option>
+        <chef-option value='roles' data-cy="roles-tab">Roles</chef-option>
+        <chef-option value='details' data-cy="details-tab">Details</chef-option>
+        </chef-tab-selector>
+      </chef-page-header>
+      <section class="page-body" *ngIf="tabValue === 'details'">
+        <form [formGroup]="updateOrgForm">
+          <chef-form-field>
+            <label>
+            <span class="label">Name <span aria-hidden="true">*</span></span>
+            <input chefInput formControlName="name" type="text" autocomplete="off"
```

Comment:
Need to add `resetOrigin` to each `<input>` -- please read description in form-control.directive.ts